packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -216,6 +218,10 @@ $sidebar-width-desktop: 324px;
 		position: fixed;
 		width: $sidebar-width;
 		padding-right: 22px;
+		max-height: calc( 100% - 60px - 24px - 3px );
```

Comment:
Maybe setting the following styles is easier to understand? But I'm okay with the current way 👍

```css
top: 60px;
bottom: 24px;
margin-top: 3px;
```

Reply:
I like this way, much easier 👍 
It has to be:

```
top: 63px;
bottom: 24px;
```